src/main/java/com/aetherteam/aether_genesis/blockTileEntity/HolystoneFurnaceMenu.java

```diff
@@ -0,0 +1,25 @@
+package com.aetherteam.aether_genesis.blockTileEntity;
+
+import com.aetherteam.aether.inventory.menu.AbstractAetherFurnaceMenu;
+import com.aetherteam.aether.inventory.menu.AetherMenuTypes;
+import net.minecraft.world.Container;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.inventory.ContainerData;
+import net.minecraft.world.inventory.FurnaceMenu;
+import net.minecraft.world.inventory.RecipeBookType;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.RecipeType;
+
+public class HolystoneFurnaceMenu extends AbstractAetherFurnaceMenu {
```

Comment:
You could probably just extend AbstractFurnaceMenu for this since it doesn't need any of the custom code from AbstractAetherFurnaceMenu.